talks/models.py

```diff
@@ -74,6 +74,9 @@ class Reply(models.Model):
 
     def __str__(self):
         return self.comment[:280] + '...'
```

Comment:
This is way too much, are you using this in the frontend?

Reply:
I did a different slice with jinja. I will reduce to 100.